### PR TITLE
Add stream md5s

### PR DESCRIPTION
**Pull request by privatezero**
Adds command to generate MD5s for video and audio streams (as discussed briefly in https://github.com/amiaopensource/ffmprovisr/pull/293)

### CODE REVIEW

**Comment by kfrn-nec** [comment]
Hey, looks good Andrew. However I would include the extension of the output file (`.md5`? Can't remember offhand).

**Comment by kfrn** [comment]
Opps - commented on my work account by accident! Anyway, it was a major reading comprehension failure, so just ignore me.

File: index.html [hunk 1]
```diff
@@ -1672,6 +1672,28 @@ <h3>Create MD5 checksums (audio samples)</h3>
     </div>
     <!-- ends Create frame md5s (audio) -->
 
+    <!-- Create stream md5s -->
+    <label class="recipe" for="create_stream_md5s">Create MD5 checksum(s) for A/V stream data only</label>
+    <input type="checkbox" id="create_stream_md5s">
+    <div class="hiding">
+      <h3>Create stream MD5s</h3>
+      <p><code>ffmpeg -i <i>input_file</i> -map 0:v:0 -c:v copy -f md5 <i>output_file1</i> -map 0:a:0 -c:a copy -f md5 <i>output_file2</i></code></p>
```
Comment (retokromer) [inline index.html]: Nit: I would write `output_file_1` and `output_file_2` for readability.

**Review by retokromer — COMMENTED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (privatezero) [reply index.html]: yeah, that makes sense!

**Comment by retokromer** [comment]
Thank you very much, you rock!

**Pull request merged**